pkg/proc/internal/ebpf/helpers.go

```diff
@@ -1,13 +1,12 @@
-//go:build ebpf
-// +build ebpf
+//go:build linux && amd64 && go1.16 && go1.17
+// +build linux,amd64,go1.16,go1.17
```

Comment:
Isn't it sufficient to have go1.17 here? Won't go1.17 always imply go1.16? Also why is go1.17 needed?

Reply:
We really just need > go1.15 because of the embed feature.